server/api/login/login.js

```diff
@@ -0,0 +1,45 @@
+import Q from 'q'
+import jsforce from 'jsforce'
+
+const username = 'howardxchen@gmail.com.blueprint'
```

Comment:
this repo is public so we probably should be extracting this from a `json` or a environment variable.